### ISSUE TITLE
How to specify database name?

### ISSUE THREAD

**Issue by jzpchen** [sequenceiq/docker-liquibase#10]
Thanks for the docker image. It is very useful. I did run into some difficulty in using database name other than the default "postgres" when doing update and generate. I looked at the scripts/liquidbase-setup.sh, I saw the following:

url: jdbc:postgresql://$DB_PORT_5432_TCP_ADDR:$DB_PORT_5432_TCP_PORT/$POSTGRES_USER

When I did "-e POSTGRES_USER=test", then it started to use database "test". Is this intended?

**Comment by Pouja** [comment]
Yes and no. The name of the variable is wrong. After the slash it is not $POSTGRES_USER but the database name. It should be name something like $DB_NAME.
